Documentation/nvme-micron-ocp-telemetry-log-parse.txt

```diff
@@ -0,0 +1,89 @@
+nvme-micron-ocp-telemetry-log-parse
+===================================
+
+NAME
+----
+nvme-micron-ocp-telemetry-log-parse - Parses OCP Telemetry DA1 and DA2 logs.
+
+SYNOPSIS
+--------
+[verse]
+'nvme micron ocp-telemetry-log-parse' <device>
+			[--telemetry-log=<file> | -l <file>]
+			[--string-log=<file> | -s <file>]
+			[--output-file=<file> | -o <file>]
+			[--format=<fmt> | -f <fmt>]
+
+DESCRIPTION
+-----------
+For the given NVMe device, parses the telemetry log and string log
+(in binary format) and provides the parsed data in json and normal text formats.
+
+The <device> parameter is mandatory and may be either the NVMe
+character device (ex: /dev/nvme0), or a namespace block device (ex:
+/dev/nvme0n1). 
+
+This will only work on Micron devices devices of model numbers 51Bx.
+Support for new devices may be added subsequently. Results for any other device
+are undefined.
+
+OPTIONS
+-------
+-l <file>::
+--telemetry-log=<file>::
+	This option will allows the users to specify input telemetry-log file name.
```

Comment:
This option will "allow", no s. 2 more "allows" below.

Reply:
Thanks for catching, corrected!